charts/seed-dns/entry/values.yaml

```diff
@@ -1,6 +0,0 @@
-name: internal
```

Comment:
Name does not get defaulted to `internal` anymore but that seems to be fine as it is set explicitly when created.

Reply:
Yes